pkg/commands/objectstore/liststore.go

```diff
@@ -29,7 +29,7 @@ func NewListCommand(parent cmd.Registerer, globals *config.Data, data manifest.D
 		},
 		manifest: data,
 	}
-	c.CmdClause = parent.Command("list", "List Fastly object stores")
+	c.CmdClause = parent.Command("list", "List object stores")
```

Comment:
I think the file name should stay as `list.go`

Reply:
Ye, missed this in my previous commit. Fixed.